Cargo.toml

```diff
@@ -41,10 +41,12 @@ spv-out = ["spirv"]
 wgsl-in = ["codespan-reporting"]
 wgsl-out = []
 hlsl-out = []
+error-reporting = ["codespan-reporting"]
 
 [[bin]]
 name = "naga"
 path = "bin/naga.rs"
+required-features = ["error-reporting"]
```

Comment:
Is this needed? If we keep this, cargo will require us to pass this feature all the time.

For example:
```
cargo run --features wgsl-in,wgsl-out test.wgsl test_gen.wgsl module.txt

warning: invalid feature `error-reporting` in required-features of target `naga`: `error-reporting` is not present in [features] section
error: target `naga` in package `naga` requires the features: `error-reporting`
Consider enabling them by passing, e.g., `--features="error-reporting"`
```